doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+=============================
+Multi-Minion Setup on Windows
+=============================
+
+There may be a scenario where having a minion running in the context of the
+current, logged-in user would be useful. For example, the normal minion running
+under the service account would perform machine-wide, administrative tasks. The
+minion running under the user context could be launched when the user logs in
+and would be able to perform configuration tasks as if it were the user itself.
+
+The steps required to do this are as follows:
+
+1. Create new root_dir
+2. Set root_dir permissions
+3. Create directory structure
+4. Write minion config
+
+We will now go through each of these steps in detail.
+
+.. note::
+
+    We have created a powershell script that will configure an additional minion
+    on the system for you. It can be found in the root of the Salt installation.
+    The script is named ``multi-minion.ps1``. You can get help on how to use the
+    script by running the following in a PowerShell prompt:
+
+    ``Get-Help .\multi-minion.ps1 -Detailed``
+
+1. Create new ``root_dir``
+--------------------------
+
+The minion requires a root directory to store config, cache, logs, etc. The user
+must have full permissions to this directory. The easiest way to do this is to
+put the ``root_dir`` in the Local AppData directory (``$env:LocalAppData``).
+
+.. code-block:: powershell
+
+    New-Item -Path "$env:LocalAppData\Salt Project\Salt" -Type Directory
+
+2. Set ``root_dir`` permissions
+-------------------------------
+
+The user running Salt requires full access to the ``root_dir``. If you have
+placed the root_dir in a location that the user does not have access to, you'll
+need to give the user full permissions to that directory.
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $User    = "<user running salt>"
+    $acl = Get-Acl -Path "$RootDir"
+    $access_rule = New-Object System.Security.AccessControl.FileSystemAccessRule($User, "Modify", "Allow")
+    $acl.AddAccessRule($access_rule)
+    Set-Acl -Path "$RootDir" -AclObject $acl
+
+3. Create directory structure
+-----------------------------
+
+Salt expects a certain directory structure to be present to avoid unnecessary
+messages in the logs. This is usually handled by the installer. Since we're
+running our own instance, we need to do it. Make sure the following directories
+are present:
```

Comment:
**Suggestion (style, blocking):** Related to my previous comment, this needs to change to the second person. My suggested fix is:

```
Since you are running your own instance, you need to do it.
```